### PR TITLE
Pass along headers if present.

### PR DESCRIPTION
**Pull request by ralphbean**
These may be useful to filters or fedmsg.meta routines.

### CODE REVIEW

File: fedmsg/consumers/ircbot.py [hunk 1]
```diff
@@ -342,6 +342,11 @@ def consume(self, msg):
         log.debug("Got message %r" % msg)
         topic, body = msg.get('topic'), msg.get('body')
 
+        # Pass along headers if present.  May be useful to filters or
+        # fedmsg.meta routines.
+        if 'headers' in msg
```
Comment (jeremycline) [inline fedmsg/consumers/ircbot.py]: You missed a colon here.

**Review by jeremycline — CHANGES_REQUESTED** [review]
There's a syntax error. I'd also really like to see test coverage for this change.

File: fedmsg/consumers/ircbot.py (hunk 1 above)
Comment (ralphbean) [reply fedmsg/consumers/ircbot.py]: Doh, thanks.  Rebased.

**Comment by ralphbean** [comment]
> I'd also really like to see test coverage for this change.

Added in 951d1aa.

**Review by jeremycline — APPROVED** [review]
Looks good, thanks!

:shipit:

**Pull request merged**